astropy/tests/tests/test_runner.py

```diff
@@ -1,9 +1,11 @@
-from astropy.tests.runner import TestRunner, TestRunnerBase, keyword
```

Comment:
I specifically avoided "Test" anywhere in the name because I can never figure out how sensitive the test discovery algorithm is to such a thing.